### PR TITLE
test: added tests for uploading documents and images.

### PR DESCRIPTION
**Pull request by inlag**
Added tests for the /api/cdn/upload/image/ and /api/cdn/upload/doc/ endpoints. (issue #69)

### CODE REVIEW

File: src/handlers/docs/handleDocUpload_test.go [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+func TestHandleDocUpload_NoError(t *testing.T) {
+	// create tmpFile
+	pipeRead, pipeWriter := io.Pipe()
+	writer := multipart.NewWriter(pipeWriter)
+	go func() {
+		defer writer.Close()
+		part, _ := writer.CreateFormFile("doc", "filename.txt")
+		part.Write(testDataFile)
+
+		//h := make(textproto.MIMEHeader)
```
Comment (kevinanielsen) [inline src/handlers/docs/handleDocUpload_test.go:29]: What are these comments?

**Review by kevinanielsen — CHANGES_REQUESTED** [review]
You've left a few comments - otherwise everything looks great 😄

File: src/handlers/docs/handleDocUpload_test.go (hunk 1 above)
Comment (inlag) [reply src/handlers/docs/handleDocUpload_test.go]: Sorry didn't notice, corrected.

**Review by kevinanielsen — APPROVED** [review]

**Comment by kevinanielsen** [comment]
Looks great! Thanks for the contribution!

**Pull request merged**